ci/dev-image/exec.sh

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+# exec.sh opens an interactive bash session inside of a docker container 
+# for improved isolation during development
```

Comment:
Could you elaborate that it reuses the container if it exists? And starts it if stopped.